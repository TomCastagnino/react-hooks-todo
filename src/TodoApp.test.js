import { shallow, mount } from 'enzyme';
import React from 'react';
import TodoApp from './TodoApp';

describe('<TodoApp />', () => {

    it('renders a list of todos', () => {
        const wrapper = mount(<TodoApp />);
        expect(wrapper).toMatchSnapshot();
        wrapper.unmount();
    });

});