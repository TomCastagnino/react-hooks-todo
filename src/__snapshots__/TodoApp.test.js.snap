// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoApp /> renders a list of todos 1`] = `ReactWrapper {}`;
